apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/style-tour.scss

```diff
@@ -0,0 +1,63 @@
+@import '~@wordpress/base-styles/colors';
+@import '~@wordpress/base-styles/mixins';
+@import '~@wordpress/base-styles/variables';
+@import '~@wordpress/base-styles/z-index';
+
+.wpcom-editor-welcome-tour-container {
+	background-color: $white;
+	border-radius: 2px;
+	bottom: 40px;
+	display: inline;
+	left: 40px;
+	position: fixed;
+}
+
+.welcome-tour-card__heading,
+.welcome-tour-card__subheading {
+	font-size: $big-font-size;
```

Comment:
I was a little suprised to not find a well defined typography system.  Is this the right approach?

I also tried to use the GB [ <Text/> component](https://github.com/WordPress/gutenberg/tree/master/packages/components/src/text), was excited about this, but ran into React errors. Brief chat about that [here](p1605165530464200-slack-CRWCHQGUB).   I should probably follow up on this to understand if it _should_ be usable....

Reply:
> I was a little suprised to not find a well defined typography system. Is this the right approach?

Do you think we need one in the ETK? Genuine question, I don't know! :) Seems as if the various plugin within the ETK are all doing their own "thing".

> I also tried to use the GB component, was excited about this, but ran into React errors. 

Sorry, this is going to sound very pithy (because it is), but "when in doubt, throw it out". 

If it's blocking you, don't bother. Add custom CSS if you need to and we can circle back if you like.

Reply:
Agreed, I just briefly played with [<Text/>](https://github.com/WordPress/gutenberg/tree/master/packages/components/src/text) component.  Briefly, because it threw errors right away. But on my list to reach out to ask questions about it's usage.

Not blocked by anything in terms of typography, but maybe worth asking some devs/designers about their opinion on typography system, or lack thereof, in ETK.

Reply:
@sixhours has been working on type systems at Calypso side, although this component is in intersection of .com and Gutenberg so things can work little different. I know base-styles package has some variables to use and I made a little attempt here to package Calypso variables as well: https://github.com/Automattic/wp-calypso/pull/43718 — feel free to take over if you see it fitting.